src/test/java/com/kruczala/michal/SquareRootTest.java

```diff
@@ -24,12 +24,12 @@ public void SquareRootByZero() {
     }
 
     @Test
-    public void SquareRootByNegativeNumber() {
+    public void C() {
         AdvanceCalculator advanceCalculator = new AdvanceCalculator();
         double numberToSquareRoot = -9;
         advanceCalculator.squareRoot(numberToSquareRoot);
         throw new IllegalArgumentException("Square Root from negative number does not exist");
-        // nie przechodzi mi ten test dalej...:
+        // nie przechodzi mi ten test dalej....jak to zrobić?:
```

Comment:
Zrobiłem Ci ostatnio takie 3 testy przy dzieleniu, w jednym z nich było właśnie sprawdzanie wyjątków w testach, sprawdź tamten kod, a gdybyś nie zrozumiał, to daj znać